### DISCUSSION TITLE
Create multi-column foreign keys

### DISCUSSION THREAD

**Discussion by ygoe** [pawelsalawa/sqlitestudio#4781] [Q&A]
### Details
I have two tables that are linked by two columns. I'd like to add a foreign key here to see it my query runs faster. SQLiteStudio can only create a foreign key for a single column to the other table, not for two columns. I'm pretty sure SQLite supports that, but I'll have to look up the SQL syntax for it. It would be nice if the UI would allow me to do that more quickly.

Update: Looks like I cannot add a foreign key to an existing table with SQLite (it's so limited it can barely do anything…), so the extra work SQLiteStudio does by creating the temporary table would be extremely useful here!

**Comment by pawelsalawa** [comment]
Use this one:
![image](https://user-images.githubusercontent.com/481846/236345943-f2df8912-c44c-4af6-baf6-fd2ecf623d3e.png)
